rest/route/stats_test.go

```diff
@@ -96,3 +96,33 @@ func (s *StatsSuite) TestReadStartAt() {
 	_, err = handler.readStartAt("1998-07-12|variant1|task1|test1")
 	s.Require().Error(err)
 }
+
+func (s *StatsSuite) TestRunTaskHandler() {
+	sc := &data.MockConnector{
+		MockStatsConnector: data.MockStatsConnector{},
+		URL:                "https://evergreen.mongodb.com/task",
```

Comment:
use example.net for this kind of thing. I want to avoid even the appearance of tests that hit prod evergreen.

Reply:
Done.